events/admin.py

```diff
@@ -17,14 +21,25 @@ class EventAdmin(TranslatableAdmin):
         "short_description",
         "capacity_per_occurrence",
         "participants_per_invite",
-        "created_at",
-        "updated_at",
+        "is_published",
     )
     list_display_links = ("id", "name")
-    exclude = ("id", "created_at", "updated_at")
+    exclude = ("id", "created_at", "updated_at", "published_at")
     inlines = [
         OccurrencesInline,
     ]
+    actions = ["publish"]
+
+    def publish(self, request, queryset):
+        guardians = Guardian.objects.all()
+        for obj in queryset:
+            obj.publish()
+            send_event_notifications_to_guardians(
+                obj, NotificationType.EVENT_PUBLISHED, guardians
+            )
+        self.message_user(request, "%s successfully published." % queryset.count())
+
+    publish.short_description = "Publish selected events"
```

Comment:
This and the above message should be marked as translatable because all of the admin UI is (or at least it should be :) )